packages/beacon-node/src/chain/validation/lightClientOptimisticUpdate.ts

```diff
@@ -53,15 +55,19 @@ export function validateLightClientOptimisticUpdate(
  * xxx|-------  (x is not okay)
  *
  * [IGNORE] The *update is received after the block at signature_slot was given enough time to propagate
- * through the network -- i.e. validate that one-third of *update.signature_slot has transpired
- * (SECONDS_PER_SLOT / INTERVALS_PER_SLOT seconds after the start of the slot, with a MAXIMUM_GOSSIP_CLOCK_DISPARITY allowance)
+ * through the network --
+ * i.e. validate that `get_slot_component_duration_ms(SYNC_MESSAGE_DUE_BPS)`
+ * milliseconds (with a `MAXIMUM_GOSSIP_CLOCK_DISPARITY` allowance) has
+ * transpired since the start of `signature_slot`.
  */
 export function updateReceivedTooEarly(
   config: ChainForkConfig,
   genesisTime: number,
   update: Pick<LightClientOptimisticUpdate, "signatureSlot">
 ): boolean {
-  const signatureSlot13TimestampMs = computeTimeAtSlot(config, update.signatureSlot + 1 / 3, genesisTime) * 1000;
-  const earliestAllowedTimestampMs = signatureSlot13TimestampMs - MAXIMUM_GOSSIP_CLOCK_DISPARITY;
+  const updateCutoff =
```

Comment:
might wanna keep `Ms` suffix
```suggestion
  const updateCutoffMs =
```